api/app/src/routes/user.ts

```diff
@@ -165,12 +189,12 @@ router.delete(
  *
  * Revoke access to all providers and permanently delete the user
  *
- * @param {string} req.query.userId      The internal user ID.
+ * @param {string} req.params.userId      The internal user ID.
  *
  * @return  {{success: boolean}}      If successfully removed.
  */
 router.delete(
-  "/delete",
+  "/:id",
   asyncHandler(async (req: Request, res: Response) => {
     const userId = getUserIdFrom("query", req).orFail();
```

Comment:
We need to update this as well. Might need to determine the path param as with a diff name depending on what `getUserIdFrom` requires.